test/support/models/thing.rb

```diff
@@ -3,6 +3,7 @@ class Thing < ActiveRecord::Base
   measured_length :length, :width
 
   measured Measured::Length, :height
+  measured Measured::Volume, :volume
```

Comment:
Could you add a test case here that covers the `measured_volume` method too? Otherwise it's not in the test suite and nothing to prevent regression.